### DISCUSSION TITLE
MTEX reconstruction

### DISCUSSION THREAD

**Discussion by wanghaijian217** [ORTools4MTEX/ORTools#53] [Q&A] [answered]
Hi Frank, 

As we all know, MTEX and ORTOOLS can reconstruct the original austenite grains, but how to reconstruct the remaining retained austenite after quenching? Especially, retained austenite will transform into twin martensite in the subsequent tempering process. How should this be analyzed? Please pay attention to the heat treatment means of ferroalloy materials during quenching and tempering.

best wishes!!!

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @wanghaijian217,

I assume you are trying to make a distinction between reconstructed austenite and retained austenite.

In this case, I'd advise you to do the following:
- Rename the fcc phase into your map as retainedAustenite. 
This is the phase that contains an indexed fcc phase fraction in your map.

- Copy and paste the above phase information to add one more fcc phase to your map.
Rename this phase as reconstructAustenite. 
This phase has no indexed points in your map.

- Now use ORTools to reconstruct PAGs.
In this case, the bcc Iron phase is the child phase and the fcc reconstructAustenite is the parent phase.
Reconstruction will be undertaken without touching the fcc retainedAustenite phase.

In this way, you can keep the 2 fcc phases separate for further downstream analysis.
NOTE: This logic can be applied regardless of ebsd map file format.

In the example given below, I am using *.cpr and *.crc files from OI Channel-5.
Here I am assuming you have 2 phases in your map:
Phase 1 = Iron bcc (old), and 
Phase 2 = Iron fcc

1. Open the *.cpr file in Notepad.

2. Add one more to the phase count 
```
[Phases]
Count=2
```

3. Rename phase 2 "Iron fcc" as "retAustensite".
```
[Phase2]
StructureName=retAustensite
Reference=Z. Angew. Phys. [ZAPHAX], vol. 23, pages 245-249
Enabled=True
a=3.6599
b=3.6599
c=3.6599
alpha=90.00
beta=90.00
gamma=90.00
LaueGroup=11
SpaceGroup=225
ID1=3803863129_5.0.0.1277
ID2=1619986839
```

4. Copy the whole section containiing the above fcc phase information, paste directly below it. 
Rename the phase number to "Phase3" and the phase name to "reconAustenite".
```
[Phase3]
StructureName=reconAustenite
Reference=Z. Angew. Phys. [ZAPHAX], vol. 23, pages 245-249
Enabled=True
a=3.6599
b=3.6599
c=3.6599
alpha=90.00
beta=90.00
gamma=90.00
LaueGroup=11
SpaceGroup=225
ID1=3803863129_5.0.0.1277
ID2=1619986839
```

5. Save the the *.cpr file.

6.  Now you have 2 options:
- You can import the *.cpr and *.crc into ORTools, or 
- Go to Channel-5, export a *.ctf and import that into ORTools instead.

7. Once you have imported the file into ORTools, begin reconstruction.
Ensure "Iron bcc (old)" is the child phase and "reconAustenite" is the parent phase.

Hope this helps.

<ins>**Lastly, and importantly, if this explanation resolves your issue, please mark this message as your answer.**</ins>


Warm regards,
Azdi
